src/main/app/claims/models/claim.ts

```diff
@@ -250,6 +261,17 @@ export class Claim {
     return this.settlement && !!this.settlementReachedAt
   }
 
+  private isCCJPaidWithinMonth (): boolean {
+    let futureMonth
+    futureMonth = calculateMonthIncrement(this.countyCourtJudgmentRequestedAt)
+
+    return this.moneyReceivedOn.isSameOrBefore(futureMonth)
```

Comment:
combine all above lines of code into this line. should be isSameOrBefore(calculateMonthIncrement(this.countyCourtJudgmentRequestedAt))